src/razor/src/extension.ts

```diff
@@ -99,17 +100,27 @@ export async function activate(
         // Save user's DOTNET_ROOT env-var value so server can recover the user setting when needed
         env.DOTNET_ROOT_USER = process.env.DOTNET_ROOT ?? 'EMPTY';
 
+        let telemetryExtensionDllPath = '';
         // Set up DevKit environment for telemetry
         if (csharpDevkitExtension) {
             await setupDevKitEnvironment(env, csharpDevkitExtension, logger);
```

Comment:
(Irrelevant to this PR), I would've assumed `setupDevKitEnvironment` would be named `setupDevKitEnvironmentAsync` instead because of the `await`. Do the naming conventions not apply on ts?

From the diff here, I couldn't tell where `csharpDevkitExtension` is defined.